vstools/functions/render.py

```diff
@@ -57,6 +57,31 @@ def clip_async_render(
     callback: Callable[[int, vs.VideoFrame], T] | None = None, prefetch: int = 0,
     backlog: int = -1, y4m: bool | None = None, async_requests: int | bool | AsyncRenderConf = False
 ) -> list[T] | None:
+    """
+    Iterate over an entire clip and optionally write results in a file.
+
+    This is mostly useful for metric gathering that must be performed before you do anything else.
+    This could be for example gathering scenechanges, per-frame heuristics, etc.
+
+    It's highly recommended to perform as little filtering as possible on the input clip for speed purposes.
+
+    :param clip:            Clip to render.
+    :param outfile:         Optional binary output to write. @@EXPAND@@
+    :param progress:        @@PLACEHOLDER@@
+    :param callback:        Callback function. Must accept `n` and `f` (like a frameeval would) and return some value.
+                            This function is used to determine what information gets returned per frame.
+                            Default: None.
+    :param prefetch:        The amount of frames to render concurrently. 0 means automatically determine.
+                            Default: 0.
+    :param backlog:         @@PLACEHOLDER@@
```

Comment:
how many frames to hold on the backlog 
say, if your write or callback is slower than frame rendering you would have a growing backlog because frames aren't used fast enough